openmdao/docs/other/om_command.rst

```diff
@@ -110,16 +118,38 @@ model obtained using the command:
 
    An example of a connection viewer.
 
+
+By default the promoted names columns of both inputs and outputs are shown, but in the example
+above, the absolute input names are shown and the promoted input names are hidden.
+
+Unconnected inputs can easily be identified by typing 'NO CONNECTION' or 'NO ', into
+the filter field of either the absolute or promoted output column.  Unconnected outputs can
```

Comment:
While it is rather unlikely, I believe "NO CONNECTION" is a valid openmdao variable name.   Maybe something like an "!" (or anything restricted in names) can be the shorthand code for showing unconnected inputs.

Reply:
I noticed when looking at the _valid_var_name function that we were not preventing names like ' ' and '\t', so I just went ahead and made both of those illegal characters.  So now 'NO CONNECTION' is no longer a valid variable name.

Reply:
I removed the restriction on spaces within a variable name and changed 'NO CONNECTIONS' to '[NO CONNECTIONS]', which is not a legal name because of the '[' and ']'.  So once people know what they're looking for, the unconnecteds can be shown by filtering on just '[' or ']'.